Formula/b/bold.rb

```diff
@@ -19,8 +19,11 @@ class Bold < Formula
   def install
     # Fix illegal instruction errors when using bottles on older CPUs.
     # https://github.com/Homebrew/homebrew-core/issues/92282
-    cpu = case Hardware.oldest_cpu
-    when :arm_vortex_tempest then "apple_m1" # See `zig targets`.
+    cpu = case Hardware.oldest_cpu # See `zig targets`.
+    # Cortex A-53 seems to be the oldest available ARMv8-A processor.
+    # https://en.wikipedia.org/wiki/ARM_Cortex-A53
+    when :armv8 then "cortex_a53"
```

Comment:
Maybe one of the targets with only v8a would be more minimal? `zig targets` shows A-53 as:
```
            .cortex_a53 = .{
                "fpao",
                "v8a",
            },
```

I think the most minimal 64-bit v8a-only option is A-35 (as A-32 is 32-bit)
```
            .cortex_a32 = .{
                "v8a",
            },
            .cortex_a35 = .{
                "v8a",
            },
            .cortex_a73 = .{
                "v8a",
            },
            .kryo = .{
                "v8a",
            },
```

Reply:
Looking at `zig targets` again, the most minimal option is X-Gene 1 which may be the only CPU without CRC:
```
            .xgene1 = .{
                "perfmon",
                "v8a",
            },
```

Not sure what chances of someone running this specific CPU is. Would be okay with baseline target having at least CRC.